sentinel/src/schedule/outbound.js

```diff
@@ -112,28 +115,14 @@ const singlePush = (taskDoc, medicDoc, infoDoc, config, key) => Promise.resolve(
       return removeConfigKeyFromTask(taskDoc, key);
     }
 
-    return outbound.send(config, key, medicDoc, infoDoc)
-      .then(() => {
-        // Worked, remove entry from queue and store infodoc that outbound service has updated
-        return removeConfigKeyFromTask(taskDoc, key)
-          .then(() => db.sentinel.put(infoDoc))
-          .catch(err => {
-            if (err.status !== 409) {
-              logger.error('Failed to save infodoc');
-              logger.error(err);
-              throw err;
-            }
-
-            // While we held the infodoc open something else wrote to it. Presume it wasn't
-            // anything task related, load a fresh version and overwrite
-            return db.sentinel.get(infoDoc._id)
-              .then(freshInfoDoc => {
-                freshInfoDoc.completed_tasks = infoDoc.completed_tasks;
-                return db.sentinel.put(freshInfoDoc);
-              });
-          });
-      });
-  }).catch(() => {
+    return outbound
+      .send(config, key, medicDoc, infoDoc)
+      // Worked, remove entry from queue and store infodoc that outbound service has updated
+      .then(() => removeConfigKeyFromTask(taskDoc, key))
+      .then(() => infodocLib.saveCompletedTasks(medicDoc._id, infoDoc));
+  })
+  .catch((err) => {
+    logger.warn(`Unable to push ${medicDoc._id} for ${key}: %eo`, err);
```

Comment:
Out of curiosity, what is the `%eo` doing here?

Reply:
Oh, thanks for noticing, this should just be %o!

Reply:
:sweat_smile: Here I was thinking this was some special JS thing I did not know about yet...